### PR TITLE
[# 111652886] Add a "By User" view for Allocations View.

### PR DESCRIPTION
**Pull request by kquet**
[Tracker #111652886](https://www.pivotaltracker.com/story/show/111652886)

### CODE REVIEW

File: lib/app/views/developers.slim [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.row#developers
+  ul.epics
```
Comment (kquet) [inline lib/app/views/developers.slim]: Refactor class names.

File: lib/app/views/_nav.erb [hunk 2]
```diff
@@ -78,7 +78,6 @@
   </ul>
 
   <ul class="nav navbar-right top-nav hidden-sm hidden-xs">
-    <li><p class="navbar-text"><i class="fa fa-fw fa-lg fa-server"></i> <%= HonestyCluster::SwarmBackend.node_count %></p></li>
```
Comment (kquet) [inline lib/app/views/_nav.erb]: NONONONO

File: lib/app/views/epics.slim [hunk 3]
```diff
@@ -19,31 +19,21 @@
...
+      .grid-item
+        li.grid-item-header = epic.to_s
         li.requestors = sorted_report[epic.to_sym][:requestors].join(', ') if sorted_report[epic.to_sym][:requestors].kind_of?(Array)
         li.user-list
-          ul.users
+          ul.grid-item-section
             - sorted_report[epic.to_sym][:developers].keys.each do |user|
               li.user = user.to_s
               li.stories_list
-                ul.stories
```
Comment (kquet) [inline lib/app/views/epics.slim]: wat

**Comment by kquet** [comment]
screen shot 2016-02-15 at 3 07 54 pm
screen shot 2016-02-15 at 3 07 31 pm
screen shot 2016-02-15 at 3 07 14 pm

**Comment by kquet** [comment]
@suwyn

**Comment by suwyn** [comment]
sort by name so I can find someone

**Comment by suwyn** [comment]
conflicts need resolving

**Comment by kquet** [comment]
@suwyn conflicts resolved.

**Pull request merged**